gapic/src/main/com/google/gapid/MainWindow.java

```diff
@@ -438,7 +441,8 @@ public MainView updateAndGet(Service.TraceType traceType) {
     HelpOnlineHelp("&Online Help\tF1", SWT.F1),
     HelpAbout("&About"),
     HelpShowLogs("Open &Log Directory"),
-    HelpLicenses("&Licenses");
+    HelpLicenses("&Licenses"),
+    HelpFileBug("File a &Bug on Github");
```

Comment:
Knowing that it is on GitHub is not relevant here I think, and this is the kind of things that we would typically forget to update if we move outside of GitHub one day, so I would avoid to mention it:
`HelpFileBug("File a &Bug");`